tests/generate-docs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
I recommend switching this to a tox file since it's easy to later combine with other tests (linting, unit tests, etc). See the [docs testenv](http://fedora-infra-docs.readthedocs.io/en/latest/dev-guide/writing-tests.html#tox) - it builds the docs using -W so warnings are treated like errors. It's then pretty easy to hook this up to CI [(fedmsg does this)](https://github.com/fedora-infra/fedmsg/blob/0211e43f84b3900d95c3ac0f81e27f3bb0359647/.travis.yml#L37) to make sure the docs stay ship-shape.

Reply:
Nice, this is all new to me too. I'll try splitting this out into a different PR, possibly along with refactoring the docs for a RTD-style format. See #72 as well.